images/diagram_flowchart.drawio

```diff
@@ -0,0 +1 @@
kjs3IY+QjwVUn0ba+GS6JATxW5QYRLt8zm70X01oVbSDnywS7p3FYhIU0nkZPoLLoJu7bWxC8IBThFlVBWoZPC3gnc/naDQxBlc2WgOu4365IFu+4QegYL2z0K1jxnREqMqT6XTy9YhOmoV3+mD/p30zIha46Z8TPpAALoEgzoZhKh35cHQW7AzCRQh19Z7mL4MOPOAaNkJKv5xRR/MWo/HR4QjZzej/eIRxtiTTkAcx+gh6AUazR8mYXyiSBE0XI5pMaudJ3cqRikq4vQYKrKnA5ddqkTs9pZHHzjv74bkrK1wxoR14hSSQc9dFzGaDPnjfRGToTlss+fTKVxsf1SVmzN1YOhgd9gwAe/XwUu5ceVISeFRRn5qWhDM37dhS+JhbFp5cz2EMNhj5AT9JenkBtMmhQ5J5lrRhV8TUPeQIOLR1Sj46+58eW2YMtXt53ynJ7721MdsLeEygn+7zLLsa6rBls9W12+e2VCS25sWeZ/tpMFGYV7FFk4OufQgGVX1W89IzH8MrSbHMGU2/8/mEKzEerYwfMiXbl0bIOaYx/uQdKwU+tMN31y61emUh4YU0LLMH7nJnJ0RnExdA6UU2mtWkyW7Lg6zSdBQaX5/G2Je/BmxZ64vJFobs5IpcXTr62U3pFOJYtgvqUPgvayYWJc6CGsfAguSmHRO0HVg5Rlw/fcbXtGkzE1Njc0qDpInCl+xFwn/dArMBIPI5+tro38iFx6EVUKaZ07dADbWk/exhI8nUlIhj5RB4yrAFfvCDGG/7r80BLIeVngoqMGHz1cjLpkaZiSGnLhikQpERg7WFfZvr0/FuAVyar0+8spmWIVz0a6md2P5MIEsJ6x8iA5Ed7W1jeULLKRW973tPJAA5kHEjTfGB1agYvwOzv4b1nFfV4u92EPnhgMYQbCzfY9PDTFI5Ml9LFfzmHyxhr83s4NyQLaI6uks83x3wl/JWbX1pYI/2NHpLQELz8UAFr2NZnuVzmO7eOd9H2sszJa/TcOXjuyB9UBQs5EKD3aE9j3IxnM0hEbXYZueJtvaR3UyOsSbRd3iHG891/XpdNNKPZmdpeZAB2ycVbpiCrzFUYnpORD1ld76UAAn3uORDIMzzsW1YnnA119cVpmCGu9mDUEv26jw+mS2cE3f/398CYDAEbNDVkJQJbkXSoRYsJvzEBv0Ei9Fa838d9N6tZoqCI0EM942Ci4Sb+anWKvs/xjPPxPh1d72TeJ34yorrZfbfEsIDdYSfjmfpoG6mbrF2fPnLNfCg5gxwtChJ6GSmgCx7qc5oO/l67OcS9e91/D50htHxi7lyHPaxD/YqDZjncL2SyEf/9LdTOwBeeERiHym86lL12jeXbjqn7xYbZdkjTTeXOoScck2OKtSYndYGTVSAJm70ZMHmsFtOg1rchDqrbR1CvMf3FccriyFnLjfiAE+o2jx5P6A8lEgsbacGAngZkx/l2ZwlViOnGbW9TLrc4yyCS9+g58W0UzR9H8Aj7gnN8MdNTwuf2LRdggA4ZzPd3WI35gFHAYDw+5x9Xj4detRieJ9R5v285TqSoNRePaR8hA2fQsZCvjc7Q0Cidm9IfDYTQGt4nDMiR6atGPwnR9aBuDbvQ2hE4OUgkmyzqO4+Ha0Hucmc3soAEwRB8WIb0rvuXmOy9CtcGqVp14tJWz73wN+xQICc0Gp+iKtJj70QaPc4PHUvKvcd9h6U79cTdgeevvBfIXLKHyPVinbzl7Y2391gtWzTqg4GBJ7wvW5FZAgdkEiEE9Lks6TF5EgWrePJDR8ive5uUBW2b7xC5yZf/HIViXaqXg4CiUvPOJquaYms6QoEsesZZKjk4OrvrzQamGoiu4QRZy+zyvUq0eSsnSwbIoKM7V4d0ay0YVxfP8Lj3hWkOtKsJ5dhAiGEcd9vjCy7LMtZaT+8v98kIc/zHGWuvp3N1TzeCug9mOZ00rVtTXzRQBEcPj7ut0fZ3rpcXdchqFRbXdtAnuHsunvEAJDpsYKXFHmVBd3qhyTRvR0vL4Ic/FV654CEKV7uTwc8/rL9PqFcsK1MPN4t/nTA0LTiQ6+EVjzp03s/HFfIEWIQTH5JkG3uOAb1XZjt0T0VRVQ/ym6cBwSMjgaLjtTe58ZuPRy86vBKhe/KvW8z86WulsxXQte4QyYvG99OH3pupGQ/35Obka4824b26E9F0w4o8Zc5Z9I1+0e3XcE+CpHT4dw4JKs0iPaEHwpWbJ4X7ZfZIWXbhntSsPjQMAAcjdD2H8aUGg673/bMMU0LFOyqX05npvrfIM7QPsR9v1+UxyulfFjYqNhrpla4sabrP38F75VyFexzMq6P9lc+XarqwpKoTp++L6L2evQvNt4xB+CsfjaeRTq1tvmKc/SeHRQ9wT12vSPNf+fDrLjQ9Rqjq9Iv9UFDPBuieBI9qsvCvfPQItkjwD0mzjZ8e/YSWIX14kJ78kw8SGlKcLM8tbuzEvzqneJCzdrbYSP/Kh8k0eCAEVTP4Rv90DkU0cE9sf3rt2dmvfCIEMVhRpsxHFv/VOcH+qld0OezxA7y3kaeMdpiSTaQw0v/qnJuhexrKCnOJv/IoaVCvcj2pzPhX53z+K6TrJ87tP/IoVsCg+o1IUhr7q3O7eqmq65D/kocOqYiCEyvCnL8dC+hF9HVwT76ldFAj/8qDE+G8rVK+YZz3V+d2ISH5SzPO/pHHKmSQ9Yquy/nMZH90jPXhnrykKcO/8kBOA+J701LsNMn+6FiXg5D+3wb8z21A+78984QQoI8k4Qs7+f4zUAZuYP4lBzABkMX8slfFT+iRMlKFnPGaurkTDX9cyEqoJ6Qs8j9H7p8j8tiHfw5YH4+P9gWvHA7496X3Y3w91n976cj/c2SbVXx0zeM1Im+M/flfHIa04DPb94Ujef7+eynSV/570/H3nvxRZPnrv16Mp+8L2d8vh2LJ95LQzLSyj7r++8+x617/9n/iGPe53qUPeMf/AQ==</diagram></mxfile>
```

Comment:
Is this all objects, or just the DLC portion?

Reply:
Just the DLC diagram.